src/components/implementation/no_interface/vkernel/micro_booter.c

```diff
@@ -0,0 +1,579 @@
+#include <stdio.h>
```

Comment:
A comment that you already know:  we don't want to replicate all of this code.

We want to break the `micro_booter` into the parts that are shared, and those that are different.  Shared should be soft-linked, and what's different should be in a separate `.c` file.